version.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/opencontainers/runtime-spec/specs-go"
+	"github.com/urfave/cli"
+)
+
+// version will be populated by the Makefile, read from
+// VERSION file of the source code.
+var version = ""
+
+// gitCommit will be the hash that the binary was built from
+// and will be populated by the Makefile
+var gitCommit = ""
+
+var versionCommand = cli.Command{
+	Name:        "version",
+	Usage:       "output the runtime version",
+	Description: `The version command outputs the runtime version.`,
+	Action:      printVersion,
+}
+
+func printVersion(context *cli.Context) (err error) {
+	if version == "" {
+		_, err = os.Stdout.WriteString("runC unknown\n")
+	} else {
+		_, err = os.Stdout.WriteString(fmt.Sprintf("runC %s\n", version))
```

Comment:
> everything is printing with a : between the name and the version

So you want one here too?  I'm just matching the current master approach:

```
runc version 1.0.0-rc3
commit: 6b574d57594aedca4bb45b679788af73ae0d65f9
spec: 1.0.0-rc5
```

but I can go either way (with or without colons) if you prefer.